### PR TITLE
2022-01-14-bert_sequence_classifier_emotion_en

### PR DESCRIPTION
**Pull request by jsl-models**

### CODE REVIEW

File: docs/_posts/murat-gunay/2022-01-14-bert_sequence_classifier_emotion_en.md [hunk 1]
```diff
@@ -0,0 +1,108 @@
...
+spark_version: 3.0
+supported: true
+article_header:
+  type: cover
+use_language_switcher: "Python-Scala-Java"
+---
+
+## Description
+
+This model was imported from `Hugging Face` and it's been fine-tuned on emotion [dataset](https://huggingface.co/nlp/viewer/?dataset=emotion), leveraging `Bert` embeddings and `BertForSequenceClassification` for text classification purposes.
```
Comment (josejuanmartinez) [inline docs/_posts/murat-gunay/2022-01-14-bert_sequence_classifier_emotion_en.md:20]: I think here the link should be to the model (nateraw/bert-base-uncased-emotion) not the dataset

File: docs/_posts/murat-gunay/2022-01-14-bert_sequence_classifier_emotion_en.md [hunk 2]
```diff
@@ -0,0 +1,108 @@
...
+|Max sentence length:|128|
+
+## Data Source
+
+[https://huggingface.co/datasets/viewer/?dataset=emotion](https://huggingface.co/datasets/viewer/?dataset=emotion)
+
+## Benchmarking
+
+```bash
+Val. Accuracy : 0.931
```
Comment (josejuanmartinez) [inline docs/_posts/murat-gunay/2022-01-14-bert_sequence_classifier_emotion_en.md:107]: Val. accuracy is not a useful metric, but I took a look and the author didn't share any other. Can you add the following?

```
NOTE: The author didn't share Precision / Recall / F1, only Validation Accuracy was shared as [Evaluation Results](https://huggingface.co/nateraw/bert-base-uncased-emotion#eval-results).

Validation Accuracy: 0.931 
```

**Review by josejuanmartinez — APPROVED** [review]
Can you apply this changes? Thank you!

File: docs/_posts/murat-gunay/2022-01-14-bert_sequence_classifier_emotion_en.md (hunk 1 above)
Comment (murat-gunay) [reply docs/_posts/murat-gunay/2022-01-14-bert_sequence_classifier_emotion_en.md:20]: @josejuanmartinez 
I changed the link

File: docs/_posts/murat-gunay/2022-01-14-bert_sequence_classifier_emotion_en.md (hunk 2 above)
Comment (murat-gunay) [reply docs/_posts/murat-gunay/2022-01-14-bert_sequence_classifier_emotion_en.md:107]: @josejuanmartinez 
I added the text you suggested.

**Review by josejuanmartinez — APPROVED** [review]
Looks good, thanks!

**Pull request merged**